src/bepasty/tests/test_rest_server.py

```diff
@@ -139,6 +139,26 @@ def unlock(self):
             return url_for('bepasty_apis.items_unlock', name=self.item_id)
 
 
+def test_secret_key(client_fixture):
+    Config.PERMISSIONS = {
+        'admin': 'admin,list,create,read,delete',
+        'full': 'list,create,read,delete',
+        'none': '',
+    }
+    Config.SECRET_KEY = 'secret'
+
+    app = create_app()
+    secret_key = app.config['SECRET_KEY']
+    assert secret_key > Config.SECRET_KEY
```

Comment:
did you really mean sort order or did you forget `len(...) > len(...)`?

Reply:
ouch, fixed